programs/aes/aescrypt2.c

```diff
@@ -87,6 +87,9 @@ int main( int argc, char *argv[] )
     unsigned char buffer[1024];
     unsigned char diff;
 
+#if defined(MBEDTLS_PLATFORM_C)
+    mbedtls_platform_context platform_ctx;
```

Comment:
In case of _ALT implementations where this context will be used by the platform specific code, it may be useful to create it global instead of a stack variable. We don't know how big it could be and if it will fit the embedded application stack. Secondly, platform specific code may want to access it as an ```extern``` object.

Reply:
@mazimkhan Do you advise doing this for all the examples?
@RonEld What do you think about such use case? It sound's reasonable, but I'd like to consult this with more reviewers?

Reply:
Yes as the argument is general.

Reply:
It seems to me that there is no immediate need to make platform context global in this PR. @mazimkhan do you agree?

Reply:
Ok if there is an issue #1200 addressing scope, memory requirement and ownership of this object then I am fine leaving it as is.